pkg/pillar/cmd/nim/nim.go

```diff
@@ -181,6 +182,7 @@ func (n *nim) init() (err error) {
 		SubEdgeNodeCert:      n.subEdgeNodeCert,
 		PubCipherBlockStatus: n.pubCipherBlockStatus,
 		CipherMetrics:        n.cipherMetrics,
+		HVTypeKube:           node.IsHVTypeKube(),
```

Comment:
Shouldn't we put IsHVTypeCube in a common package? Having cmd/X import cmd/Y will over time result in loops.

Reply:
Done